src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -2073,13 +2090,13 @@ internal static string ConcatenateStringPathArguments(CommandElementAst stringAs
                     }
                 case "Show-Command":
                     {
-                        NativeCompletionGetHelpCommand(context.WordToComplete, parameterName, false, result, context);
+                        NativeCompletionGetHelpCommand(context, parameterName, false, result);
```

Comment:
When there are many places where we use constants or an enum, it might be nicer to use a constant instead of named parameters, e.g.

```C#
const bool helpRelatedFalse = false;
const bool helpRelatedTrue = true;
NativeCompletionGetHelpCommand(context, parameterName, helpRelatedFalse, result);
// or
enum HelpRelated { True, False }
NativeCompletionGetHelpCommand(context, parameterName, HelpRelated.False, result);
```